index.html

```diff
@@ -99,6 +109,93 @@ <h2>Multi-Post <br> Stories </h2>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+        <div class="desk-second-div">
+            <div class="div-info">
+                <h2>Multi-Post Stories</h2>
+                <ul class="ul-li">
+                    <li>FACEBOOK</li>
+                    <li><img src="images/Counter.png"></li>
+                    <li>Full Stack Dev</li>
+                    <li><img src="images/Counter.png"></li>
+                    <li>2015</li>
+                </ul>
+                <p>Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
+                <ul class="sec-list">
+                    <li>html</li>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="desk-div-img"><img src="images/Snapshoot Portfolio (2).png"></div>
+        </div>
+        <div class="desk-third-div">
+            <div class="desk-div-img"><img src="images/Snapshoot Portfolio (3).png"></div>
+            <div class="div-info">
+                <h2>Facebook 360</h2>
+                <ul class="ul-li">
+                    <li>FACEBOOK</li>
+                    <li><img src="images/Counter.png"></li>
+                    <li>Full Stack Dev</li>
+                    <li><img src="images/Counter.png"></li>
+                    <li>2015</li>
+                </ul>
+                <p>Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
+                <ul class="sec-list">
+                    <li>html</li>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+        <div class="desk-fourth-div">
+            <div class="desk-div-img"><img src="images/Snapshoot Portfolio.png"></div>
+            <div class="div-info">
+                <h2>Uber Navigation</h2>
+                <ul class="ul-li">
+                    <li>Uber</li>
+                    <li><img src="images/Counter.png"></li>
+                    <li>Back End Dev</li>
+                    <li><img src="images/Counter.png"></li>
+                    <li>2015</li>
+                </ul>
+                <p>A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+                <ul class="sec-list">
+                    <li>html</li>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
```

Comment:
- [ ]  Please 🙏 adjust the tech stack of your cards, and align them to the right to make it look like the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=Z0umLtHjW2jvzRBK-0)

**_Figma_**

<img width="125" alt="Screenshot 2023-02-23 at 10 40 15 PM" src="https://user-images.githubusercontent.com/44798044/220986696-8b05f633-f6da-4abc-bbb1-0df54fbabc20.png">

**_Yours_**

<img width="309" alt="Screenshot 2023-02-23 at 9 21 29 PM" src="https://user-images.githubusercontent.com/44798044/220986730-2d7dfa04-f64e-4f90-b98a-b17d940708ab.png">